src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
```

Comment:
100 is a magic value , refer to the checklist

Reply:
not fixed, magic numbers must be constants, don't forget to use CONSTANT_CASE for their names